### PR TITLE
Run rubocop -a and tweak a few cop settings

### PR DESCRIPTION
**Pull request by johnnyshields**

### CODE REVIEW

**Comment by johnnyshields** [comment]
Please merge

**Comment by CloCkWeRX** [comment]
Most of the time, this is safe; but such wide sweeping changes can accidentally introduce bugs and end up invalidating a lot of existing PRs...

With Growstuff, we tackled it with incremental PRs and it led to refactoring; ie https://github.com/Growstuff/growstuff/pull/1216 or https://github.com/Growstuff/growstuff/pull/1401

I'm kind of -1 to doing it in one big hit

**Comment by johnnyshields** [comment]
@CloCkWeRX please read the actual changes one-by-one I would not expect this to cause issues. If you can comment on specific changes that you'd like rolled back I'd be happy to do so.

**Comment by johnnyshields** [comment]
Have extracted out two PRs which account for the lion's share of code changes. Will rebase this once those are merged.

**Comment by johnnyshields** [comment]
@CloCkWeRX rebased, the code is more digestable now. Can you please review item by item and give feedback?

File: .rubocop.yml [hunk 1]
```diff
@@ -6,15 +6,42 @@ AllCops:
...
+  EnforcedStyle: prefer_alias
+
+Style/EmptyMethod:
+  EnforcedStyle: expanded
+
 Style/MutableConstant:
   Enabled: false
 
+Style/SafeNavigation:
+  Enabled: false
```
Comment (CloCkWeRX) [inline .rubocop.yml]: #647 for later

**Comment by CloCkWeRX** [comment]
Might be worth merging master in again for this

**Comment by johnnyshields** [comment]
Will do after we merge all the outstanding PRs related to rubocop changes (string literal etc.)

**Comment by johnnyshields** [comment]
I've raised 3 more Rubocop PRs, when those are merged I will rebase this PR.

**Comment by johnnyshields** [comment]
Replaced by #681

**Pull request closed without merging**